picky/src/jose/jwk.rs

```diff
@@ -371,9 +375,22 @@ impl Jwk {
                     }),
                 }
             }
-            SerdePublicKey::Ed(_) => Err(JwkError::UnsupportedAlgorithm {
-                algorithm: "edwards curves",
-            }),
+            SerdePublicKey::Ed(_) => {
```

Comment:
*Sigh*... Missed it in the previous PR. Now it is covered by ECDH tests

Reply:
Thank you!